reports/2022.html

```diff
@@ -799,51 +803,116 @@ <h3>Links</h3>
           <dt>Previous WCCG Report(s)</dt>
           <dd><a href="https://w3c.github.io/webcomponents-cg/index.html#declarative-custom-elements">2021</a></dd>
           <dt>GitHub issues:</dt>
-          <dd>---</dd>
+          <dd>
+            <ul>
+              <li><a href="https://github.com/WICG/webcomponents/blob/gh-pages/proposals/Declarative-Custom-Elements-Strawman.md">Initial Proposal</a></li>
+              <li><a href="https://github.com/Westbrook/custom-element-proposals/pull/1">Single File Component Explainer</a></li>
+              <li><a href="https://github.com/WICG/webcomponents/issues/645">Cross-over with HTML Modules</a></li>
+            </ul>
+          </dd>
           <dt>Browser positions:</dt>
           <dd>---</dd>
         </dl>
       </section>
       <section>
         <h3>Description</h3>
-        <p>---</p>
+        <p>A method of creating custom elements completely with declarative HTML, not requiring JavaScript.</p>
       </section>
       <section>
         <h3>Status</h3>
         <ul>
-          <li>---</li>
+          <li>Strawperson proposal available.</li>
+          <li>Dependent on many other proposals that are not yet finished.</li>
+          <li>Many people would like to have something like this but it seems too early due to the number of unfinished dependencies.</li>
         </ul>
       </section>
       <section>
         <h3>Initial API Summary/Quick API Proposal</h3>
-        <p>Summary or proposal based on current status; paragraph(s) and code.</p>
+        <p>The following, copied from the original strapwerson proposal, demonstrates how a declarative custom element could be defined with the need for JavaScript.</p>
+        <pre class="html">&lt;definition&nbsp;name=&quot;percentage&#45;bar&quot;&gt;
```

Comment:
@EisenbergEffect 
I think this might be what is causing the build failure?
https://github.com/w3c/webcomponents-cg/runs/7933450202?check_suite_focus=true
```sh
Run w3c/spec-prod@v2
Run actions/setup-node@v2
Found in cache @ /opt/hostedtoolcache/node/16.16.0/x64
Run echo "::group::Set up action"
Set up action
Run echo "::group::Prepare"
Prepare
Run echo "::group::Setup toolchain"
Setup toolchain
Run echo "::group::Generate Static HTML"
Generate Static HTML
  [INFO] Build respec document "reports/2022.html" (common)…
    $ respec -s "http://localhost:3000/reports/2022.html" -o "reports/2022.html.built.html" --verbose -t 20 -e
      [INFO] [Timeout: 20000ms] Processing resource: http://localhost:3000/reports/2022.html ...
      [INFO] [Timeout: 19998ms] Launching browser
      [INFO] [Timeout: 19536ms] Navigating to http://localhost:3000/reports/2022.html
      [INFO] [Timeout: [18](https://github.com/w3c/webcomponents-cg/runs/7933450202?check_suite_focus=true#step:3:20)631ms] Navigation complete.
      [INFO] [Timeout: 18596ms] Using ReSpec v32.2.3
      [INFO] [Timeout: 18596ms] Processing ReSpec document...
  Error: ROR] Couldn’t find “root“ in this document or other cited documents: [dom], [fetch], [html], [infra], [url], and [webidl].
         Count: 1
        Plugin: core/xref
          Hint: [See search matches for “root”](https://respec.org/xref/?term=root&types=_IDL_) or [Learn about this error](https://respec.org/docs/#error-term-not-found).
  Error: ROR] Couldn’t find “attributes“, for “root”,  in this document or other cited documents: [dom], [fetch], [html], [infra], [url], and [webidl].
         Count: 1
        Plugin: core/xref
          Hint: [See search matches for “attributes”](https://respec.org/xref/?term=attributes&for=root&types=attribute%2Cdict-member%2Cconst) or [Learn about this error](https://respec.org/docs/#error-term-not-found).
  Error: ROR] Couldn’t find “percentage“, for “attributes”,  in this document or other cited documents: [dom], [fetch], [html], [infra], [url], and [webidl].
         Count: 1
        Plugin: core/xref
          Hint: [See search matches for “percentage”](https://respec.org/xref/?term=percentage&for=attributes&types=attribute%2Cdict-member%2Cconst) or [Learn about this error](https://respec.org/docs/#error-term-not-found).
  Error: ROR] Couldn’t find “value“, for “percentage”,  in this document or other cited documents: [dom], [fetch], [html], [infra], [url], and [webidl].
         Count: 1
        Plugin: core/xref
          Hint: [See search matches for “value”](https://respec.org/xref/?term=value&for=percentage&types=attribute%2Cdict-member%2Cconst) or [Learn about this error](https://respec.org/docs/#error-term-not-found).
  Error: ROR] Couldn’t find “percentage“ in this document or other cited documents: [dom], [fetch], [html], [infra], [url], and [webidl].
         Count: 1
        Plugin: core/xref
          Hint: [See search matches for “percentage”](https://respec.org/xref/?term=percentage&types=_IDL_) or [Learn about this error](https://respec.org/docs/#error-term-not-found).
      [INFO] [Timeout: 18085ms] Processed document.
      [INFO] [Timeout: 1[707](https://github.com/w3c/webcomponents-cg/runs/7933450202?check_suite_focus=true#step:3:716)8ms] Done.
      [FATAL] Error: Errors found during processing.
          at run (/home/runner/work/_actions/w3c/spec-prod/v2/node_modules/respec/tools/respec2html.js:247:11)
          at processTicksAndRejections (node:internal/process/task_queues:96:5)
    Failed
  Error: Process completed with exit code 1.
Run # Failed
Error: Failed. See details above.
Error: Process completed with exit code 1.
```

Reply:
Yeah. I’m not sure the right way to put “code” into this tool, but it definitely seems to be looking at this like JS instead of HTML representation of JS.

Reply:
@EisenbergEffect I'm sure there is a better answer out there, but in the here and now, I suggest you replace all of the `{{` usage here in with `{​{` or `:curly_brace::no_width_space::curly_brace:` to encourage the parse not to assume that the code you are exhibiting here is meant to be compiled at render time.